packages/sfui/frameworks/react/components/SfInput/SfInput.tsx

```diff
@@ -4,9 +4,9 @@ import { SfInputSize, useFocusVisible, type SfInputProps, polymorphicForwardRef
 const defaultWrapperTag = 'span';
 
 const sizeClasses = {
-  [SfInputSize.sm]: 'py-1.5 px-4',
-  [SfInputSize.base]: 'py-2 px-4',
-  [SfInputSize.lg]: 'py-3 px-4',
+  [SfInputSize.sm]: ' h-[32px]',
+  [SfInputSize.base]: 'h-[40px]',
+  [SfInputSize.lg]: 'h-[48px]',
```

Comment:
Cool, but now you removed `py` which is different for each size. I think you should restore it.

Reply:
oh so I probably misunderstood this suggestion:  I think it'd be better to set a fixed height (e.g. h-[32px]) and remove vertical padding (py).

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2714/commits/fa0a8ee2972239d55b7cc0a5e1626d0d19e80589

Reply:
Yes! You're absolutely right! I'm so sorry, may bad. We set the height and remove the vertical padding.